src/main.rs

```diff
@@ -133,6 +135,11 @@ fn audit(lockfile_path: String, verbose_output: bool) -> ! {
     }
 }
 
+fn banner() {
+    println!("{}", std::include_str!("banner.txt"));
```

Comment:
I'm on the fence about always showing the banner. Our other CLI apps tend to only show a banner with in "loud"/verbose mode, as users expressed a preference for "quiet by default". It's not a blocker for me, but maybe showing the banner only when the "loud"/v flag is present is a good compromise? Your call.

Either way, I really like the idea of always showing the name/version text, regardless of flags.

Reply:
ah ok cool, I didnt realise that was how it currently works on others. The raised issue didn't make that statement. I will update 👍

Reply:
https://github.com/sonatype-nexus-community/cargo-pants/pull/26/commits/a9e7c190443fffe5639e594bd696c37465931193